pkg/config/config.go

```diff
@@ -219,6 +221,7 @@ func setDefaults(v *viper.Viper) {
 	v.SetDefault("options.introspect_api_time_limit_sec", DefaultIntrospectApiTimeLimitSec)
 	v.SetDefault("options.always_run_cron_tasks", false)
 	v.SetDefault("options.enable_notifications", false)
+	v.SetDefault("options.enable_templates_notifications", false)
```

Comment:
we should probably default this to true.  We set notifications to false as we didn't want it turned on in production.   In fact, i might would argue we make the topic configurable (defaulting to 'platform.content-sources.template') and send it if its set.    That way, you're implicitly turning it on by setting the topic to deliver it on.